### ISSUE TITLE
Rinex 3.02 errors in gettime 

### ISSUE THREAD

**Issue by neibegafig** [geospace-code/georinex#77]
Noticing a timing issue with the gettime function for rinex 3.02 files. They have not just a header, but also a footer so it seems, which also starts with a key that begins with ">" however is mostly empty. Example: your standard time in rinex is something like

"> 2021  01 01  01  01  "

Etc.

Rinex 3.02 files have a line like this

">                            4  68"

The numbers aren't always 4 or 68, but sometimes. A way I find around it is to modify the function to do a check where if it sees it start with that example above, without those numbers, skip it entirely for the gettime function. I can send some files if it helps clarify issue.
Basically if it's missing year, month, day, not to add in the output.
